release/README.md

```diff
@@ -2,7 +2,11 @@
 
 This repository contains the docker image for odfe-pipes.
 
-## Running a build
+## Running a docker build
 
-To build an image check out the corresponding branch for the version and run `./gradlew :release:docker`
+To build an image, check out the corresponding branch for the version and run `./gradlew clean :release:docker -Prelease`
```

Comment:
What happens if you don't run with `-Prelease`?

Reply:
the release sub project tasks will not be included - This will keep our ci builds and local builds fast